core/templates/dev/head/pages/library/SearchBarDirective.js

```diff
@@ -43,6 +43,68 @@ oppia.directive('searchBar', [
               }
             )
           );
+          $scope.activeMenuName = '';
+          $scope.ACTION_OPEN = 'open';
+          $scope.ACTION_CLOSE = 'close';
+          $scope.KEYBOARD_EVENT_TO_KEY_CODES = {
```

Comment:
This looks like a duplicate dict. can we move this to a constant (maybe oppia.constant)?